src/recommendationservice/Dockerfile

```diff
@@ -29,4 +29,5 @@ RUN python -m pip install grpcio-tools==1.48.2
 RUN python -m grpc_tools.protoc -I=./proto/ --python_out=./ --grpc_python_out=./ ./proto/demo.proto
 
 EXPOSE ${RECOMMENDATION_SERVICE_PORT}
-ENTRYPOINT [ "opentelemetry-instrument", "python", "recommendation_server.py" ]
+
+ENTRYPOINT [ "./docker-entrypoint.sh" ]
```

Comment:
To clarify, what problem are we solving exactly by changing the entrypoint like this?

Reply:
I did this just for better local development (I can install my local sentry sdk in the `docker-entrypoint.sh`

Reply:
Ok, I see. Still, since it was more of a one-off case (changing Python SDK in parallel with the service code here), let's revert the changes in Dockerfile and `docker-entrypoint` (both in `quoteservice` and `recommendationservice`) 🙏  

I feel like they create an unnecessary divergence with the upstream, and not really necessary in the long term, wdyt?

Reply:
ok

Reply:
done.